NExtends/Primitives/Strings/String.extensions.cs

```diff
@@ -0,0 +1,564 @@
+        {
+            return value.Replace("'", "''");
+        }
+
+        public static String ToFirstUpper(this String word)
+        {
+            if (!String.IsNullOrEmpty(word))
+            {
+                return word[0].ToString().ToUpper() + word.Substring(1);
+            }
+            else
+            {
+                return word;
+            }
+        }
+
+        public static String JSONProtect(this String value)
+        {
+            if (value != null)
+            {
+                String json = value.Replace("\r\n", "<br>");
+                json = json.Replace("\n", "<br>");
+                json = json.Replace("\r", "<br>");
+
+                json = json.Replace("\\", "\\\\");
+
+                json = json.Replace("'", "\\'");
+                json = json.Replace("\"", "\\\"");
+
+                return json;
+            }
+            else
+            {
+                return "";
+            }
+        }
+
+        public static String JSONQuotesProtect(this String value)
+        {
+            if (value != null)
+            {
+                String json = value.Replace("\r\n", "<br>");
+                json = json.Replace("\n", "<br>");
+                json = json.Replace("\r", "<br>");
+
+                json = json.Replace("\\", "\\\\");
+
+                json = json.Replace("\"", "\\\"");
+
+                return json;
+            }
+            else
+            {
+                return "";
+            }
+        }
+
+        //Quelqueq méthodes pour convertir des types basiques en JSON
+        public static String ToJSON(this bool b)
+        {
+            return b.ToString().ToLower();
+        }
+        public static String ToJSON(this String s)
+        {
+            return JsonConvert.SerializeObject(s);//  "\"" + s.JSONQuotesProtect() + "\"";
+        }
+        public static String ToDoubleQuotesJSON(this String s)
+        {
+            return JsonConvert.SerializeObject(s);//return "\"" + s.JSONQuotesProtect() + "\"";
+        }
+        public static String ToDoubleQuotesJSON(this int i)
+        {
+            return i.ToJSON();
+        }
+        public static String ToDoubleQuotesJSON(this double d)
+        {
+            return d.ToJSON();
+        }
+        public static String ToDoubleQuotesJSON(this DateTime d)
+        {
+            return d.ToJQuery();
+        }
+        public static String ToDoubleQuotesJSON(this bool b)
+        {
+            return b.ToJSON();
+        }
+        public static String ToDoubleQuotesJSON(this object o)
+        {
+            if (o != null)
+            {
+                if (o.GetType() == typeof(int))
+                {
+                    return ((int)o).ToDoubleQuotesJSON();
+                }
+                else if (o.GetType() == typeof(double))
+                {
+                    return ((double)o).ToDoubleQuotesJSON();
+                }
+                else if (o.GetType() == typeof(DateTime))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the 'is' operator instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS2219)